style.css

```diff
@@ -0,0 +1,94 @@
+body {
+font-family: Roboto;
+}
+.wrapper {
+width: 80%;
+margin: auto;
+}
+.header {
+display: flex;
+justify-content: space-between;
+}
+.header__logo {
+padding: 40px 0;
+font-family: PT Serif;
+font-weight: bold;
+font-size: 24px;
+line-height: 31px;
+position: relative;
+padding-left: 44px;
+}
+.logo__circle {
+position: absolute;
+left: 0;
+top: 50%;
+transform: translateY(-50%);
+width: 28px;
+height: 28px;
+}
+.logo__circle_color_dark-blue {
+background: #0056ec;
+border-radius: 50%;
+opacity: .7;
+}
+.logo__circle_color_blue {
+ left: 8px;
+ top: 56%;
+  background: #c8dbfb;
+  opacity: .7;
+  border-radius: 50%;
+}
+.menu {
+  display: flex;
+  align-items: center;
+}
+.menu__item {
+  padding: 0 13px;
+  text-decoration: none;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 21px;
+  color: #000;
+}
+ .menu__item:last-child {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)